client/my-sites/stats/activity-log/threat-alert.jsx

```diff
@@ -0,0 +1,61 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActivityIcon from '../activity-log-item/activity-icon';
+import FoldableCard from 'components/foldable-card';
+
+export class ThreatAlert extends Component {
+	state = {
+		isExpanded: false,
+	};
+
+	render() {
+		const { moment, threat, translate } = this.props;
+
+		return (
+			<FoldableCard
+				clickableHeader
+				compact
+				header={
+					<Fragment>
+						<ActivityIcon activityIcon="notice" activityStatus="error" />
+						<span>{ translate( 'Malware code detected' ) }</span>
+					</Fragment>
+				}
+			>
+				{ threat.description }
+				{ threat.files &&
+					threat.files.map( file => (
+						<Fragment key={ file.name }>
+							<p>{ file.name }</p>
+							<p>{ file.dirpath }</p>
+							<p>
+								{ moment
+									.duration( moment( file.firstDetected ).diff( moment() ) )
+									.humanize( true ) }
```

Comment:
Can we use `TimeSince` here? (https://wpcalypso.wordpress.com/devdocs/design/time-since)

Reply:
TIL - 🙇

Reply:
actually we'll have to end up switching it out since that's not how we want to display these I don't think. at least, @keoshi had chosen a week-based view for some reason, which `humanize()` gives us. I'm going to table this for now considering that and I promise to iterate later once we get more clarity if `TimeSince` is appropriate or not. thanks for pointing me to that!

Reply:
Good point. I think it's also worth making the point that using `TimeSince` will give us more consistency about this kind of thing across the product, which IMO is preferable unless we have a good case against it.

Reply:
That'd be up to @keoshi. I don't think the week breakpoint makes sense here. We'd need data from VaultPress but I think the point here is to communicate relative age and some items will probably go back pretty far, in which case number of weeks is a reasonable value.

Thanks again - I'm pretty sure I should be using `TimeSince` in other places where I'm not.